SUMO-254673 doc changes for v 1 to v 2 migrations apps (#5019)
SumoLogic/sumologic-documentation: docs/integrations/containers-orchestration/activemq.md

Configuring log and metric collection for the ActiveMQ App includes the following tasks:

### Step 1: Fields in Sumo Logic

The following [fields](/docs/manage/fields/) will always be created automatically as a part of the app installation process:

* `pod_labels_component`
* `pod_labels_environment`
* `pod_labels_messaging_system`
* `pod_labels_messaging_cluster`

> If you're using ActiveMQ in a non-Kubernetes environment, these additional fields will get created automatically as a part of app installation process:

Reviewer: ```suggestion
If you're using ActiveMQ outside a Kubernetes environment, the following additional fields will be automatically created during the app installation process:
```

    annotations:
          tailing-sidecar: sidecarconfig;data:/opt/activemq/data/activemq.log
    ```

>    1. Make sure that the ActiveMQ pods are running and annotations are applied by using the command:

Reviewer: ```suggestion
   1. Ensure that the ActiveMQ pods are running and annotations are applied by using the command:
```

</Tabs>

## ActiveMQ Monitors

import CreateMonitors from '../../reuse/apps/create-monitors.md';

<CreateMonitors/>

> There are limits to how many alerts can be enabled

Reviewer: ```suggestion
There are limits to how many alerts can be enabled.
```
